pkg/provisioning/service.go

```diff
@@ -80,7 +80,7 @@ func (s *Service) Init(ctx context.Context) error {
 	var successPls []string
 	var allPls []string
 	for _, file := range files {
-		provPipelines, all, err := s.provisionConfigFile(ctx, file)
+		provPipelines, all, err := s.provisionConfigFile(ctx, file, successPls)
```

Comment:
It would be helpful if there was a comment explaining the difference between `allPls` and `all`.